check_stability.py

```diff
@@ -353,6 +353,53 @@ def get_files_changed():
             for item in files[:-1].split("\0")]
 
 
+def get_affected_testfiles(files_changed):
+    affected_testfiles = []
+    ignore_dirs = ["conformance-checkers", "docs"]
+    manifest_file = os.path.join(os.path.abspath(os.curdir), "w3c", "web-platform-tests", "MANIFEST.json")
+    with open(manifest_file, "r") as fh:
+        manifest_contents = fh.read()
+    for changedfile_pathname in files_changed:
+        repo_root = os.path.abspath(os.path.join(os.path.abspath(os.curdir), "w3c", "web-platform-tests"))
+        changed_file_repo_path = changedfile_pathname[len(repo_root):]
+        if changed_file_repo_path in manifest_contents:
```

Comment:
That won't work when we change the manifest format, and it's kind of a hack now.

I think you want to read the manifest using the `manifest` module, and , outside this loop, do something like:

    files_changed = set(files_changed)
    for item in manifest:
        if item.path in files_changed:
            files_changes.remove(item.path)